### ISSUE TITLE
`creact-react-app` and browserslist settings

### ISSUE THREAD

**Issue by sebastianfrey** [Esri/jsapi-resources#339]
I am reporting an issue with 

- [ ] TypeScript definitions
- [x] another resource in this repository

In the [create-react-app](https://github.com/Esri/jsapi-resources/tree/master/esm-samples/jsapi-create-react-app) example is currently pointed out that in order to get `@arcgis/core` working with `cra` projects, `browserslist` configuration should might be set to:

```
{
  "browserslist": {
    "production": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
}
```

While this seems applicable to developing or prototyping applications, it feels a bit wobbly rolling out an production app with those settings.

Are there any further investigations made by your team, how to get `@arcigs/core` running with `cra` using the default `browserslist` settings without eject?

**Comment by odoe** [comment]
Just tested this out with a fresh CRA project and everything works for a production build as expected. It looks like they changed something in the babel config. We can update the README, and leave the browerlist settings as an option if the issue arises again.

**Comment by odoe** [comment]
Updated in this PR https://github.com/Esri/jsapi-resources/pull/340

**Comment by sebastianfrey** [comment]
@odoe Thanks for looking into it.

I submited this issue, because I changed `browserslist` config to 

```
[
    ">0.2%",
    "not dead",
    "not op_mini all"
]
```

an I was getting massive errors from compiled `@arcgis/core` modules  like the following one:

```
TypeError: Cannot convert undefined or null to object
    at getPrototypeOf (<anonymous>)
```

After you said it works with a fresh CRA project, I looked deeper into it and it seems that the errors where caused by an old version of `@babel/plugin-transform-classes` in my node_modules folder. The related issue was resolved here https://github.com/babel/babel/pull/12544.

In order to get everything working I had to:

1. Nuke my node_modules
2. Get rid of my exisiting yarn.lock file (*This was important, in previous attempts I kept it*)
3. Do a fresh install `yarn install`

Now everything works like a charm. Thanks again for your assistence on that.
